releasenotes/notes/subexperiment-gen-speedup-41a4e8679353d1d9.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
```

```suggestion
fixes:
```
It's not really something you need to know when upgrading; it's just that performance is all around better now.  IMO this is the more appropriate section.